monkey/monkey_island/cc/services/repository_service.py

```diff
@@ -0,0 +1,40 @@
+from monkey_island.cc.repository import IAgentConfigurationRepository, IFileRepository
+
+
+class RepositoryService:
```

Comment:
The name is rather general. Maybe a better name would be something like `ConfigurationService`?